impeller/toolkit/interop/paragraph_style.cc

```diff
@@ -28,6 +28,7 @@ void ParagraphStyle::SetFontSize(double size) {
 
 void ParagraphStyle::SetHeight(double height) {
   style_.height = height;
+  style_.has_height_override = (height != 0.0);
```

Comment:
Not sure why this isn't an optional. There is also a similar pattern around struts.